feelpp/feel/feelvf/evaluator.hpp

```diff
@@ -891,7 +891,7 @@ BOOST_PARAMETER_FUNCTION(
 
     int indexmin = 0;
     int indexmax = 0;
-    double mine = 1e-30;
+    double mine = 1e+30;
     double maxe = 1e-30;
```

Comment:
and here  `std::numeric_limits<double>::epsilon()`

Reply:
@vincentchabannes @prudhomm Actually, I think `std::numeric_limits<double>::lowest()` would even be better. Otherwise, I think the `max` might also be wrong if we want to compute the max of a negative field on multiple partitions, no ?

Reply:
Indeed, I took a look quickly this morning and I was quite surprised by the use of absolute value for the max (which explains the use of lowest positive value but I didn't take the time to understand). I think you are more inside that me to understand this issue